contracts/GenArt721CoreV3.sol

```diff
@@ -47,8 +47,14 @@ contract GenArt721CoreV3 is ERC721, Ownable, IGenArt721CoreContractV3 {
 
     // All financial functions are stripped from struct for visibility
     mapping(uint256 => address payable) public projectIdToArtistAddress;
-    mapping(uint256 => address payable) public projectIdToAdditionalPayee;
-    mapping(uint256 => uint256) public projectIdToAdditionalPayeePercentage;
+    mapping(uint256 => address payable)
+        public projectIdToAdditionalPayeePrimarySales;
+    mapping(uint256 => uint256)
+        public projectIdToAdditionalPayeePrimarySalesPercentage;
+    mapping(uint256 => address payable)
+        public projectIdToAdditionalPayeeRoyalties;
+    mapping(uint256 => uint256)
+        public projectIdToAdditionalPayeeRoyaltiesPercentage;
```

Comment:
Readability nit:

Would prefer a naming convention here of `projectIdToAdditionalPayeePrimarySales` / `projectIdToAdditionalPayeeSecondarySales` + `projectIdToAdditionalPayeePrimarySalesPercentage` / `projectIdToAdditionalPayeeSecondarySalesPercentage` for readability, I think making the two as parallel as possible makes them more idiomatic.

Reply:
realized I missed another place where we had "Royalties" instead of "SecondarySales" in a function, updated and fixed.